tests/checkup_test.go

```diff
@@ -129,7 +129,7 @@ func setupCheckupPermissions() {
 	DeferCleanup(func() {
 		err = client.RbacV1().RoleBindings(testNamespace).Delete(
 			context.Background(),
-			testKiagnoseConfigMapAccessRoleBindingName,
+			kiagnoseConfigMapAccessRoleBinding.Name,
```

Comment:
why not also use the object's namespace?
```
err = client.RbacV1().RoleBindings(kiagnoseConfigMapAccessRoleBinding.Namespace).Delete(...)
```

Reply:
Done.